run.py

```diff
@@ -834,9 +834,8 @@ def fetch_test_details(var) -> dict:
                     clients=clients,
                 )
             except BaseException:  # noqa
-                if post_to_report_portal:
-                    rp_logger.log(message=traceback.format_exc(), level="ERROR")
-
+                # setting logger for ReportPortal
+                log.set_rp_logger(rp_logger)
```

Comment:
Can we do this setting when we are processing the arguments instead of here

Reply:
done